compiler/src/iree/compiler/Dialect/Flow/IR/test/dispatch_ops.mlir

```diff
@@ -117,14 +117,14 @@ func.func @inplaceTypeChange(%arg0: tensor<4x?xf32>) -> tensor<?x4xf32> {
 // CHECK-LABEL: @region
 // CHECK-SAME: (%[[ARG0:.+]]: tensor<?x?xf32>)
 func.func @region(%arg0: tensor<?x?xf32>) -> tensor<?x?xf32> {
-  // CHECK: %[[R:.*]] = flow.dispatch.region -> (tensor<?x?xf32>{%{{.*}}, %{{.*}}}) {
+  // CHECK: %[[R:.*]] = flow.dispatch.region {operand_segment_sizes = array<i32: 2, 0>} -> (tensor<?x?xf32>{%{{.*}}, %{{.*}}}) {
```

Comment:
It is ugly to print the `operand_segment_sizes`. Maybe the parser/printer needs to be fixed to avoid this attribute?

Reply:
Good point, I did not I can handle this in parser/printer. I implemented that.